WordPress/openverse
# Update media moderation view

## Fixes



Fixes #3637 by @sarayourfriend

## Description


This PR updates the reporting view in line with the implementation plan requirements.

The media moderation view is split into 3 sections:
- current report details
- media item details
- other reports for the media item

The IP says to use `TabularInline` for the other reports, however, it's not possible to do so because a `TabularInline` can be used for displaying the child models on the parent admin page[^1]. The other reports for the media item, on the other hand, display the children of the parent of the admin page model (Child (report) -> Parent (media) -> Children (reports)). That's why I used a separate function to build a table of other reports.

[^1]: https://docs.djangoproject.com/en/5.0/ref/contrib/admin/#inlinemodeladmin-objects
## Screenshots
![image](https://github.com/WordPress/openverse/assets/15233243/a8f11869-8a4c-4703-b2ae-e19530d60fc9)


## Testing Instructions

Run the app using `just up`.
Go to `http://localhost:50280/admin` and log in using the local credentials (user: `deploy` user, password: `deploy`)
Add several reports for images and audio. Add several reports for the same item.
Then click on change report. You should see the updated form that contains a blurred image or an audio, as well as other parts of the report.
Check that add report/change report/view report views work correctly.

## Checklist



[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
